.github/workflows/qiita-plugin-ci.yml

```diff
@@ -80,11 +80,11 @@ jobs:
           conda activate qp-woltka
           # bowtie2==2.5.0 is what's installed in main qiita so making sure we keep
           # using the same version for consistency; note that we will need to force
-          conda install --yes -c bioconda bowtie2==2.5.0 seqkit
+          conda install --yes -c bioconda bowtie2==2.5.0 seqkit git
 
+          pip --quiet install -U pip nose flake8
+          pip --quiet install git+https://github.com/qiita-spots/qiita_client.git#egg=qiita_client
```

Comment:
What is the purpose of using #egg?

Reply:
To be honest, no idea but that's what the error says to use to fix it.

Reply:
ok.